test/integration_test/kubevirt_hyperconv_test.go

```diff
@@ -951,3 +993,59 @@ func getVMINameFromVMPod(vmPod *corev1.Pod) (string, error) {
 	}
 	return vmiRef.Name, nil
 }
+
+// returns a list of kubevirtTestState objects for given node
+func getTestStatesForNode(t *testing.T, ctxs []*scheduler.Context, nodeName string, allNodes map[string]node.Node) []*kubevirtTestState {
+	testStatesForNode := make([]*kubevirtTestState, 0)
+	for _, appCtx := range ctxs {
+		testState := &kubevirtTestState{
+			appCtx:   appCtx,
+			allNodes: allNodes,
+		}
+		gatherInitialVMIInfo(t, testState)
+		verifyInitialVMI(t, testState)
```

Comment:
we can call verifyInitialVMI() only if the node is the current node

Also, (for future) we don't need full "gatherInitialVMIINfo()" for all the VMs. We could first get the VM pod and check if it is running on the current node and call `gatherInitialVMIInfo` only for the VMs running on this node. But we can do this change later if gatherInitialVMIInfo() starts taking too much time.

Reply:
Moved `verifyInitialVMI` to only if node is the current node.